pkg/services/ngalert/api/api_provisioning.go

```diff
@@ -517,3 +522,43 @@ func exportResponse(c *contextmodel.ReqContext, body definitions.AlertingFileExp
 	}
 	return r(http.StatusOK, body)
 }
+
+func exportHcl(download bool, body definitions.AlertingFileExport) response.Response {
+	resources := make([]hcl.Resource, 0, len(body.Groups)+len(body.ContactPoints)+len(body.Policies))
+	for idx, group := range body.Groups {
+		gr := group
+		resources = append(resources, hcl.Resource{
+			Type: "grafana_rule_group",
+			Name: fmt.Sprintf("rule_group_%04d", idx),
+			Body: &gr,
+		})
+	}
+
+	// TODO implement support.
+	// for idx, cp := range ex.ContactPoints {
```

Comment:
Yuri, are you planning to do this TODO before merging this feature or will it come later?

Reply:
It will be addressed in a follow up PR